test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -589,28 +597,153 @@ Describe "TabCompletion" -Tags CI {
             $res.GetNextResult($true).CompletionText | Should Be 'Cmdlet'
             $res.GetNextResult($true).CompletionText | Should Be 'Configuration'
         }
+
+        It "Test history completion" {
+            $history = [pscustomobject]@{
+                CommandLine = "Test history completion"
+                ExecutionStatus = "Stopped"
+                StartExecutionTime = "8/11/2017 11:04:10 AM"
+                EndExecutionTime = "8/11/2017 11:05:49 AM"
```

Comment:
The dates is not explicitly used but it is culture dependent. Should our tests work under different cultures?